receiver/splunkhecreceiver/internal/mocks/ack_extension.go

```diff
@@ -0,0 +1,109 @@
+// Code generated by mockery v2.42.1. DO NOT EDIT.
+
+package mocks
```

Comment:
not fond of using mocks here. Can we either locate the mock as a test struct in the extension or not use a mock?

Reply:
removed